ChangeLog

```diff
@@ -77,6 +77,8 @@ Release date: TBA
 
 * Improve check if class is subscriptable PEP585
 
+* Fix documentation and filename handling of --import-graph
```

Comment:
I think we're going to release 2.7.3 before merging this change. You'll have to rebase and move this in the 2.7.4 notes later.

Reply:
You fixed the MR faster than we released, in 2.7.3 it goes :)

Reply:
:)